src/Erl/Data/Map.purs

```diff
@@ -108,6 +126,18 @@ foreign import keys :: forall a b. Map a b -> List a
 
 foreign import union :: forall k v. Map k v -> Map k v -> Map k v
 
+foreign import unionWithImpl :: forall k v. (Fn2 v v v) -> Map k v -> Map k v -> Map k v
+
+-- | Compute the union of two maps, using the specified function
+-- | to combine values for duplicate keys.
+unionWith :: forall k v. (v -> v -> v) -> Map k v -> Map k v -> Map k v
+unionWith f m1 m2 =
+  unionWithImpl (mkFn2 f) m1 m2
+
+-- | Compute the union of a collection of maps. Keeps the last value for conflicting keys.
+unions :: forall k v f. Foldable f => f (Map k v) -> Map k v
```

Comment:
well here's a thing, `Data.Map` union prefers the left/first and `Erl.Data.Map` prefers the right/last (for `union` as well as the new `unions`

Reply:
Oh, that's quite bad. Do we think anyone is relying on this? Can we change it now? We're for relying on `Data.Map.union` being left-biased in a few places atm

Reply:
I'd probably suggest first merging this PR and then making a breaking-changes PR that we let sit for at least a few days cooling off period..

Reply:
Sounds good to me

Reply:
Maybe it's a good idea to add some tests for checking the bias of the unions. Maybe add this information about the preference clearly in the documentation as well since I have definitely missed this information.

Reply:
There's tests for `unions` bias https://github.com/purerl/purescript-erl-maps/pull/10/files#diff-71732b478b4808898d86c8591ad7ab46d8122c1e4facec4a9151ac49efba905dR150-R165 and https://github.com/purerl/purescript-erl-maps/pull/10/files#diff-71732b478b4808898d86c8591ad7ab46d8122c1e4facec4a9151ac49efba905dR143-R165 but yeah, the `union` test doesn't test bias and it probably should